tasks/07_install_timer.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Create borgbackup timer
+  when:
+    - borgmatic_timer is defined and borgmatic_timer == "systemd"
```

Comment:
Would be more readable to include the sub-playbook by variable name? Then this one task could be part of another playbook.

```
      ansible.builtin.include_tasks:
        file: noauto_install_timer_{{ borgmatic_timer }}.yml
```

Reply:
I agree, it also looks better.

Done